src/app.dump.ts

```diff
@@ -1,5 +1,6 @@
 import { EnvKeys } from "hooks/useShellEnv.ts"
-import { flatmap, print } from "utils"
+import { usePrint } from "hooks";
```

Comment:
omit `;` for consistency (I know the auto-complete likes to add them).

Reply:
I decided to overlook these as we've all done it.